File: app/src/main/java/com/commit451/gitlab/model/Contributor.java
package com.commit451.gitlab.model;

import com.google.gson.annotations.SerializedName;

import org.parceler.Parcel;

@Parcel
public class Contributor {
    @SerializedName("name")
    String mName;
    @SerializedName("email")
    String mEmail;
    @SerializedName("commits")
    int mCommits;
    @SerializedName("additions")
    int mAdditions;
    @SerializedName("deletions")
    int mDeletions;

    public Contributor() {}

    public String getName() {
        return mName;
    }

    public String getEmail() {
        return mEmail;
    }

    public int getCommits() {
        return mCommits;
    }

    public int getAdditions() {
        return mAdditions;
    }

    public int getDeletions() {
        return mDeletions;
    }
}

File: app/src/main/java/com/commit451/gitlab/model/api/Contributor.kt
package com.commit451.gitlab.model.api

import android.os.Parcelable
import com.squareup.moshi.Json
import kotlinx.android.parcel.Parcelize

@Parcelize
data class Contributor(
    @Json(name = "name")
    var name: String? = null,
    @Json(name = "email")
    var email: String? = null,
    @Json(name = "commits")
    var commits: Int = 0,
    @Json(name = "additions")
    var additions: Int = 0,
    @Json(name = "deletions")
    var deletions: Int = 0
): Parcelable
